bundles/org.openhab.binding.mqtt.homeassistant/src/test/java/org/openhab/binding/mqtt/homeassistant/internal/component/ClimateTests.java

```diff
@@ -452,13 +452,6 @@ public void testClimateWithEmptyName() {
 
         assertChannel(component, Climate.ACTION_CH_ID, "living-room-tion-3s/climate/living-room-tion-3s/action/state",
                 "", "living-room-tion-3s", TextValue.class);
-        assertChannel(component, Climate.CURRENT_TEMPERATURE_CH_ID_DEPRECATED,
```

Comment:
these assertions should remain, just remove the `_DEPRECATED` from the channel ids

Reply:
done, thanks.